pkgs/development/compilers/rust/rustc.nix

```diff
@@ -11,6 +12,12 @@ let
 
   llvmShared = llvm_7.override { enableSharedLibraries = true; };
   buildLlvmShared = buildPackages.llvm_7.override { enableSharedLibraries = true; };
+
+  llvm-config-host = writeShellScriptBin "llvm-config" ''
+    exec ${buildLlvmShared}/bin/llvm-config "$@" \
+      | sed 's|${buildLlvmShared}|${llvmShared}|g' \
+      | sed 's|${buildLlvmShared.lib}|${llvmShared.lib}|g'
```

Comment:
Ah. Exactly what I proposed! I think we should make this part of llvm itself.
As it can be useful in other places.

Reply:
would I add something like `wrapLlvmConfig` to the `llvm` directory in `pkgs/development/compilers/llvm` and then add it to `pkgs/top-level/all-packages.nix`? or is there a more conventional way to add a 'global function' rather than a package?

Reply:
I am not sure about the best way to deal with this. Ideally, users who are not aware of cross-compiling should not have to take care of this when adding llvm to a project. @Ericson2314 how do you dealt with this in the past?